src/docker/Dockerfile

```diff
@@ -183,20 +195,23 @@ RUN asdf plugin add python \
 # NOTE: PIPENV_VERSION is an env var that is used by pipenv, so it can't be used here. ref: https://github.com/pypa/pipenv/issues/3633#issuecomment-478250721
 ARG PIP_ENV_VERSION="2020.11.15"
 ENV PIP_ENV_VERSION=${PIP_ENV_VERSION}
+# hadolint ignore=DL3042
```

Comment:
I don't think we want this ignore here.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3042)